docs/tutorials/synthesis.md

```diff
@@ -0,0 +1,231 @@
+> Refer to [Quickstart](./quickstart.md) for installation instructions.
+
+### Connect to the public API
+``` py
+import cript
+
+host = "criptapp.org"
+token = "<your_api_token>"  
+cript.API(host, token)
+```
+!!! note
+    Your API token can be found in the UI under [Account Settings](https://criptapp.org/settings/).
+
+
+### Create a Project node
+``` py
+proj = cript.Project(name="<your_project_name>")
+proj.save()
+```
+!!! note
+    Project names are globally unique.
+
+
+### Create a Collection node
+``` py
+coll = cript.Collection.create(project=proj, name="Synthesis Tutorial")
+```
+!!! note
+    Notice the use of `create()` here, which instantiates and saves the object in one go.
+
+### Create an Experiment node
+``` py
+expt = cript.Experiment(
+    collection=coll, 
+    name="Anionic Polymerization of Styrene with SecBuLi"
+)
+expt.save()
+```
+
+### Get Material nodes
+For this tutorial, we will get an existing Inventory node from the database.  
+This contains all of the Material nodes we will be using.
+``` py
+uid = "134f2658-6245-42d8-a47e-6424aa3472b4"
+inv = cript.Inventory.get(uid=uid, get_level=1)
+```
+!!! note
+    We are setting `get_level` to `1` so that the Material nodes are auto-generated. This parameter defaults to `0`, but can be set to any integer.
+
+Notice that the Material node objects have been auto-generated.
+``` py
+type(inv.materials[0])
+# <class 'cript.data_model.nodes.material.Material'>
+```
+
+### Create a Process node
+``` py
+prcs = cript.Process(
+    experiment=expt, 
+    name="Anionic of Styrene",
+    type = "multistep",
+    description = "In an argon filled glovebox, a round bottom flask was filled with 216 ml of dried toluene. The "
+                  "solution of secBuLi (3 ml, 3.9 mmol) was added next, followed by styrene (22.3 g, 176 mmol) to "
+                  "initiate the polymerization. The reaction mixture immediately turned orange. After 30 min, "
+                  "the reaction was quenched with the addition of 3 ml of methanol. The polymer was isolated by "
+                  "precipitation in methanol 3 times and dried under vacuum."
+)
+prcs.save()
+```
+
+### Add Ingredient nodes to the Process node
+First, let's grab the Material nodes we need from the Inventory node.
+``` py
+solution = inv['SecBuLi solution 1.4M cHex']
```

Comment:
I don't think inventory indexing works at the movement with chemical names.
See Issue #53